src/parameterizations/lateral/MOM_self_attr_load.F90

```diff
@@ -148,52 +162,90 @@ subroutine calc_love_scaling(nlm, rhoW, rhoE, Love_Scaling)
 
   do m=0,nlm ; do n=m,nlm
     l = order2index(m,nlm)
-    Love_Scaling(l+n-m) = (3.0 / real(2*n+1)) * (rhoW / rhoE) * (1.0 + KDat(n+1) - HDat(n+1))
+    if (CS%use_bpa) then
+      CS%Love_scaling(l+n-m) = (3.0 / real(2*n+1)) * (1.0 / (rhoE * grav)) * (1.0 + KDat(n+1) - HDat(n+1))
```

Comment:
We have the chance here to avoid an extra division (and the need for a future answer date) by defining a variable like `I_g_rhoE = 1.0 / (rhoE * grav)` outside of the `do m` and `do n` loops.  Please consider making this change.

  Unfortunately for `(rhoW / rhoE)` it is already too late to avoid the division without changing answers, so that one should probably stay as it is.

Reply:
@Hallberg-NOAA This subroutine is only called during initialization, which is why I did not bother to pre-calculate the divisions. (If we use n=40, this translates to ~1600 divisions). 

But I think for good practice purposes, it should be modified. Does the parenthesis around `rhoW/rhoE` shield potential answer changes if we swap it with a pre-calculated variable?

Reply:
Yes, I do think that the parentheses around `(rhoW/rhoE)` would allow us to precalculate it without changing answers.